internal/graph/check.go

```diff
@@ -314,44 +337,76 @@ func exclusion(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHa
 		wg.Done()
 	}()
 
+	var dbReads uint32
 	for i := 0; i < len(handlers); i++ {
 		select {
 		case baseResult := <-baseChan:
 			if baseResult.err != nil {
-				return &openfgapb.CheckResponse{Allowed: false}, baseResult.err
+				return &ResolveCheckResponse{
```

Comment:
If there is an error why do we need to return this? And in case we do, can we make this a var first and return it in the case of err or !allowed?